admin/js/wizard.js

```diff
@@ -272,7 +291,7 @@ function saveConfig(config) {
       }
   console.log('saving settings: ', config);
   showMessage(`Setting option: ${option.key}`);
-  sendApiRequest('/option', option, 'disciple-tools-setup-wizard/v1')
+  sendApiRequest('/option', option, 'dt-admin-settings')
```

Comment:
This cause the settings screen to not actually updated the config value correctly. Can't we just use the `setOption()` function that is defined above? That should be cleaner than manually sending another separate API request